content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -168,14 +168,36 @@ The kubelet provides a gRPC service to enable discovery of dynamic resources of
 running Pods. For more information on the gRPC endpoints, see the
 [resource allocation reporting](/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/#monitoring-device-plugin-resources).
 
-## Limitations
-
-The scheduler plugin must be involved in scheduling Pods which use
-ResourceClaims. Bypassing the scheduler by setting the `nodeName` field leads
-to Pods that the kubelet refuses to start because the ResourceClaims are not
-reserved or not even allocated. It may be possible to [remove this
-limitation](https://github.com/kubernetes/kubernetes/issues/114005) in the
-future.
+## Pre-scheduled Pods
+
+When creating a Pod with `nodeName` already set, the scheduler gets bypassed.
+If some ResourceClaim needed by that Pod does not exist yet, is not allocated
+or not reserved for the Pod, then the kubelet will fail to run the Pod and
+re-check periodically because those requirements might still get fulfilled
+later.
+
+Such a situation can also arise when support for dynamic resource allocation
+was not enabled in the scheduler at the time when the Pod got scheduled
+(version skew, configuration, feature gate, etc.). kube-controller-manager
+detects this and tries to make the Pod runnable by triggering allocation and/or
+reserving the required ResourceClaims.
+
+However, it is better to avoid this because a Pod which is assigned to a node
+blocks normal resources (RAM, CPU) that then cannot be used for other Pods
+while the Pod is stuck. To make a Pod run on a specific node while still going
+through the normal scheduling flow, create the Pod with a node selector that
```

Comment:
```suggestion
through the normal scheduling flow, create the Pod with a `nodeSelector` that
```

Not quite sure if this is correct but you capitalize other api fields or used code style for those.  Thought it looked a bit off.

Reply:
"a node selector" refers to the general concept here, not the specific field. That is then shown in the example. I think using plain English is fine here and also used elsewhere.